tests/quadsim/robots/test_robots.py

```diff
@@ -80,6 +81,7 @@ def test_reset(on_rack, hard_reset):
 def test_step():
     pybullet_client, sim_conf = setup()
     robot = A1(pybullet_client, sim_conf=sim_conf)
+    motor_command = MotorCommand(desired_position=robot._motor_group._init_positions)
     for _ in range(10):
-        robot.step(robot._motor_group._init_positions)
```

Comment:
good call here. The previous argument `robot._motor_group._init_positions` was an awful thing that never should have been written (if you git blame you'll see I was the offender XD).

Reply:
While I'm thinking about it, we should be careful to not access any 'protected' variables throughout our code. I think I have a separate issue related to this to look into.

Reply:
Thanks, fixed.
Regarding protected access, I'm okay with protected access in tests, but not in actual code.

Again, it would be great if flake8 could catch this.

Reply:
Yeah I wonder why that isn't a part of `flake-8-bugbear`. I will take a look!v